catalyst/contrib/dl/callbacks/__init__.py

```diff
@@ -1,4 +1,5 @@
 # flake8: noqa
 
+from .cutmix_callback import CutmixCallback
 from .knn import KNNMetricCallback
-from .metricaggregatorcallback import MetricAggregatorCallback
+from .metric_aggregator_callback import MetricAggregatorCallback
```

Comment:
btw, speaking about naming, there is a good questions: do we really need callback postfix for .py files?
why not to use just `cutmix.py` and `metric_aggregator.py`?